.github/workflows/continuous-deployment.yml

```diff
@@ -345,6 +369,7 @@ jobs:
     outputs:
       cosmosDbClientId: ${{ steps.azure-db-deploy.outputs.cosmosDbClientId }}
       cosmosDbPrincipalId: ${{ steps.azure-db-deploy.outputs.cosmosDbPrincipalId }}
+      vnetName: ${{ steps.azure-db-deploy.outputs.cosmosDbPrincipalId }}
```

Comment:
This doesn't seem right. Do we need this line?

Reply:
This can be removed. Fortunately, we use the ref for vnetName correctly within this job when creating slot resources for the webapp. On line 455